### PR TITLE
Create ravenkeeper

### PR DESCRIPTION
**Pull request by McTsts**

### CODE REVIEW

**Review by rupertandrosy — APPROVED** [review]
looks good.

File: limited/ravenkeeper [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-Instead of assigning a raven, the Ravenkeeper can also use their ravens to spy. Using the ravens to spy is an immediate effect. For the rest of the night, the Ravenkeeper will be told when a player with a raven uses an action on another player with a raven. The Ravenkeeper will be told the names of both players, and who used the action, but not what sort of action was done.
+The Ravenkeeper starts out with 1 raven for every 5 players in the game (rounded down). Each night, the Ravenkeeper may assign one raven to another player. Ravens can also be reassigned from one player to another (ravens that were assigned to dead players will immediately return to the Ravenkeepr upon the pplayer's death). Assigning a raven is an immediate effect. Ravens not assigned to another player are considered to be assigned to the Ravenkeeper.
```
Comment (12Metatron) [inline limited/ravenkeeper:5]: "Ravenkeepr upon the pplayer's death" is a typo

Comment (rupertandrosy) [reply limited/ravenkeeper:5]: luffy im not the only one who can fix typos you know. you could have done that yourself.

**Comment by rupertandrosy** [comment]
we sure this should be TI and not TP?

**Comment by McTsts** [comment]
Hm yea, kinda is TP

**Comment by rupertandrosy** [comment]
consider changing that then

**Pull request merged**